### PR TITLE
Feature/loading component

### PR DESCRIPTION
**Pull request by patriciafemenias**

### CODE REVIEW

File: src/components/Loading/Loading.test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      render(
+        <CharactersProviderWrapper>
+          <ThemeProvider theme={mainTheme}>
+            <Loading />
+          </ThemeProvider>
+        </CharactersProviderWrapper>,
+      );
+
+      const gif = screen.getByAltText(expectedAltText);
+      expect(gif).toBeInTheDocument();
```
Comment (fjgrande) [inline src/components/Loading/Loading.test.tsx:22]: Separar el actions del assert

**Review by adricruzado — CHANGES_REQUESTED** [review]
Good job Trizia! Check the structure of the test (no space in line 21) and there's a little mistake in line 8. Thank you!

**Review by fjgrande — CHANGES_REQUESTED** [review]
Favor revisar comentarios

**Review by adricruzado — APPROVED** [review]

**Review by fjgrande — APPROVED** [review]
Buen trabajo!

**Pull request merged**